packages/geo/src/lib/print/shared/print.service.ts

```diff
@@ -650,10 +707,11 @@ export class PrintService {
       }).then( e => {
         canvasOverlayHTML = e;
       });
+
+      // reset canvas transform to initial
+      context.setTransform(1, 0, 0, 1, 0, 0);
       context.drawImage(canvasOverlayHTML, 0, 0);
-      if (olCollapsed) {
-        element.classList.add('ol-collapsed');
-      }
+      document.getElementById('print-area').remove();
```

Comment:
On pourrait supprimer l'élément en utilisant directement la référence `mapOverlayHTML`. Ainsi on aurrait plus de besoin de référencer nos éléments dom avec des id.
```suggestion
      this.removeDomElement(mapOverlayHTML);
      
      private removeDomElement(element: HTMLElement): void {
        if (!element.parendNode) { return; }
        element.parentNode.removeChild(element);
      }
```